### ISSUE TITLE
Build error on Ubuntu 14 X86_64

### ISSUE THREAD

**Issue by reeze** [dotnet/coreclr#131]
```
Commencing CoreCLR Repo build
Checking pre-requisites...
Commencing build of native components for amd64/debug
Invoking cmake with arguments: /home/vagrant/coreclr DEBUG
Detected Linux x86_64
-- Configuring done
-- Generating done
-- Build files have been written to: /home/vagrant/coreclr/binaries/CMake
Executing make install -j 2
[  0%] Built target cppmunge
[  1%] Built target palrt
[  7%] Built target CoreClrPal
[  7%] Built target gcinfo
[  8%] Built target mdcompiler_wks
[  9%] Built target mdruntime_wks
[  9%] Built target mdruntimerw_wks
[  9%] Built target mdhotdata_full
[  9%] Built target ceefgen
[ 10%] Built target ildbsymlib
[ 10%] Built target cordbee_wks
[ 11%] Built target corguids
[ 11%] Built target strongname_wks
[ 13%] Built target v3binder
[ 13%] Built target bcltype
[ 13%] Built target comfloat_wks
[ 16%] Built target utilcode_dac
[ 19%] Built target utilcodestaticnohost
[ 19%] Building ASM object src/vm/wks/CMakeFiles/cee_wks.dir/__/amd64/jithelpers_fastwritebarriers.S.o
/tmp/jithelpers_fastwritebarriers-650074.s:314:21: error: invalid hexdecimal number
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:314:21: error: unknown token in expression
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:323:21: error: invalid hexdecimal number
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:323:21: error: unknown token in expression
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:357:21: error: invalid hexdecimal number
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:357:21: error: unknown token in expression
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:366:20: error: invalid hexdecimal number
        movabs r8, 0F0F0F0F0F0F0F0F0h
                   ^
/tmp/jithelpers_fastwritebarriers-650074.s:366:20: error: unknown token in expression
        movabs r8, 0F0F0F0F0F0F0F0F0h
                   ^
/tmp/jithelpers_fastwritebarriers-650074.s:374:21: error: invalid hexdecimal number
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:374:21: error: unknown token in expression
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:464:21: error: invalid hexdecimal number
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
/tmp/jithelpers_fastwritebarriers-650074.s:464:21: error: unknown token in expression
        movabs rax, 0F0F0F0F0F0F0F0F0h
                    ^
make[2]: *** [src/vm/wks/CMakeFiles/cee_wks.dir/__/amd64/jithelpers_fastwritebarriers.S.o] Error 1
make[1]: *** [src/vm/wks/CMakeFiles/cee_wks.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 25%] Built target cee_dac
```

```
vagrant@vagrant-ubuntu-trusty-64:~/coreclr$ llvm-
llvm-ar              llvm-config          llvm-dis-3.4         llvm-mc              llvm-ranlib          llvm-stress-3.4
llvm-ar-3.4          llvm-config-3.4      llvm-dwarfdump       llvm-mc-3.4          llvm-ranlib-3.4      llvm-symbolizer-3.4
```

**Comment by shahid-pk** [comment]
you need llvm and clang 3.5

**Comment by reeze** [comment]
@shahid-pk Thank you I will try 3.5.

**Comment by jkotas** [comment]
Will be addressed by #132
